client/extensions/woocommerce/app/settings/shipping/shipping-labels.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldSet from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSelect from 'components/forms/form-select';
+import FormToggle from 'components/forms/form-toggle';
+import ShippingCard from './shipping-card';
+import ShippingHeader from './shipping-header';
+
+class ShippingLabels extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state with real data
+		this.state = {
+			visible: true,
+			cards: [ {
+				selected: true,
+				type: 'VISA',
+				digits: '1234',
+				name: 'Name Surname',
+				date: '12/19'
+			}, {
+				selected: false,
+				type: 'MasterCard',
+				digits: '5678',
+				name: 'Name Surname',
+				date: '01/20'
+			} ]
+		};
+
+		this.onToggle = this.onToggle.bind( this );
```

Comment:
Here's a good article I found just yesterday about this: http://egorsmirnov.me/2015/08/16/react-and-es6-part3.html#method-4-using-fat-arrow-function-and-es2015-class-properties

Always in motion, the codebase is :)